mne/commands/utils.py

```diff
@@ -4,14 +4,29 @@
 #
 # License: BSD (3-clause)
 
-import imp
+import sys
 import os
 import re
 from optparse import OptionParser
 
 import mne
 
 
+def load_module(name, path):
+    if sys.version_info < (3, 3):
+        import imp
+        return imp.load_source(name, path)
```

Comment:
My guess is that these get byte-compiled while building for installation, which means this won't work for `pyc` files even if it works during dev. One of the Travis builds runs from an installation directory so hopefully will tell us, but it might be safer to put the old conditional back for completeness.